_rules/video-audio-described-1ea59c.md

```diff
@@ -13,6 +13,10 @@ input_aspects:
 authors:
   - Wilco Fiers
   - Brian Bors
+htmlHintIgnore:
+ # https://www.npmjs.com/package/htmlhint 
+ # (used with `npm test` to ensure validity of code snippets)
+ - 'tag-pair'
```

Comment:
What is this one for specifically?

Reply:
See their Wiki - https://github.com/htmlhint/HTMLHint/wiki/Tag-pair, and good catch, I removed some wrong closing tags for `<source>` element.